WcaOnRails/lib/database_dumper.rb

```diff
@@ -569,6 +569,8 @@ def self.development_dump(dump_filename)
         populate_table_sql = "INSERT INTO #{dump_db_name}.#{table_name} (#{column_sanitizers.keys.join(", ")}) SELECT #{column_expressions} FROM #{table_name} #{table_sanitizer[:where_clause]}"
         ActiveRecord::Base.connection.execute(populate_table_sql)
       end
+
+      ActiveRecord::Base.connection.execute("INSERT INTO #{dump_db_name}.timestamps (name, date) VALUES ('developer_dump_exported_at', UTC_TIMESTAMP())")
```

Comment:
1. Yeah, that is wired up to our real database, whereas I want to write into the `wca_development_db_dump` database.
2. Good call! I've made that change.